### PR TITLE
Only consider main branch in README test badge

### LINKED ISSUE #384: Test badge in Readme is not showing last test on main
### Description

It's directing to https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests

Which is the most recent PR test?

Should probably be https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests+branch%3Amain

### PR DESCRIPTION
**Pull request by Prtm2110**
The README badge now shows if the latest tests on the main branch passed. 

## Related Issue

- [x] Closes #384
- [ ] Related to #384

## Checklist

- [ ] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

### CODE REVIEW

**Comment by twiecki** [comment]
Thanks!
Isn't it using the main branch by default anyway?

**Comment by ricardoV94** [comment]
> Thanks!
> Isn't it using the main branch by default anyway?

Nope, it was using the most recent tests, usually from a draft PR.
At least it was doing that when I opened the original issue

File: README.rst [hunk 1]
```diff
@@ -134,6 +134,6 @@ A good place to start contributing is by looking through the issues
 
 .. |Project Name| replace:: PyTensor
 .. |Tests Status| image:: https://github.com/pymc-devs/pytensor/workflows/Tests/badge.svg
-  :target: https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests
+  :target: https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests+branch%3Amain
```
Comment (ricardoV94) [inline README.rst:137]: This seems to include main from forks: https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests+branch%3Amain

**Comment by maresb** [comment]
@Prtm2110, could you please fix the title of this PR?

File: README.rst [hunk 2]
```diff
@@ -134,6 +134,6 @@ A good place to start contributing is by looking through the issues
 
 .. |Project Name| replace:: PyTensor
 .. |Tests Status| image:: https://github.com/pymc-devs/pytensor/workflows/Tests/badge.svg
```
Comment (ricardoV94) [inline README.rst:136]:
```suggestion
.. |Tests Status| image:: https://github.com/pymc-devs/pytensor/workflows/Tests/badge.svg?branch=main
```

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
Thanks @Prtm2110

**Comment by Prtm2110** [comment]
Thanks @ricardoV94 for my first PR

**Pull request merged**